src/components/validations/validation-icon.component.js

```diff
@@ -81,6 +87,8 @@ ValidationIcon.propTypes = {
   iconId: PropTypes.string,
   /** Define position of the tooltip */
   tooltipPosition: PropTypes.string,
+  /** Overrides the default flip behaviour of the Tooltip, must be an array containing some or all of ["top", "bottom", "left", "right"] (see https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements) */
+  tooltipFlipOverrides: PropTypes.array,
```

Comment:
I think you should add the propType validation here as well rather than allowing any array

Reply:
Fair point, I was weighing up whether we wanted three errors (ValidationIcon -> Icon -> Tooltip) to be thrown when an invalid array is passed, but if you think it's worth it I'll add it in here and in `Help`

Reply:
Yeah I see what you mean but for the sake of total clarity for the user I think it should be the component they try to use that emits an error if the prop type is incorrect, even if it's children emit errors as well

If we ever convert the project to TypeScript it would be so much better for stuff like this!